projects/igniteui-angular/src/lib/grid/grid.component.ts

```diff
@@ -129,6 +129,11 @@ export interface IColumnMovingEndEventArgs {
     cancel: boolean;
 }
 
+export interface IGroupingDoneEventArgs {
```

Comment:
This is insufficient for the user to determine the state change. At the very least it should be:

```typescript
export interface IGroupingDoneEventArgs {
     expressions: Array<ISortingExpression> | ISortingExpression;
     ungroupedColumns: columns: Array<IgxColumnComponent> | IgxColumnComponent;
     groupedColumns: columns: Array<IgxColumnComponent> | IgxColumnComponent;
}
```

This is because the user can push new grouping expressions that result in ***both*** grouping and ungrouping in the same operation.

Reply:
In the docs we should also be clear that these are only the ***newly*** grouped and ***newly*** ungrouped columns. The columns that changed their grouping state.